js/editor-libs/console-utils.js

```diff
@@ -57,7 +57,7 @@ module.exports = {
             }
         }
 
-        if (objectName === 'Symbol') {
+        if (objectName === 'Symbol' && (typeof input != 'undefined')) {
```

Comment:
Because we are using `'use strict';` so we can safely check against `undefined`. Also use `!==`

```
if (objectName === 'Symbol' && (typeof input !== undefined)) {
```

Reply:
Ok - good points :)